<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBild_laden"/>
    <addaction name="actionEnde"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <widget class="QMenu" name="menuFilter">
     <property name="title">
      <string>Filter</string>
     </property>
     <addaction name="actionKantenfilter_Canny"/>
     <addaction name="actionLaPlace_Operator"/>
     <addaction name="actionSobelfilter"/>
     <addaction name="actionGaussfilter"/>
     <addaction name="actionMedianfilter"/>
     <addaction name="actionMittelwertfilter"/>
     <addaction name="actionseparierter_Mittelwertfilter"/>
    </widget>
    <addaction name="actionTestbild_langsam"/>
    <addaction name="actionTestbild_schnell"/>
    <addaction name="separator"/>
    <addaction name="menuFilter"/>
    <addaction name="separator"/>
    <addaction name="actionInvertieren"/>
    <addaction name="actionHistogrammausgleich"/>
    <addaction name="separator"/>
    <addaction name="actionScaling"/>
    <addaction name="actionRotieren"/>
    <addaction name="actionDilatieren"/>
    <addaction name="actionErodieren"/>
    <addaction name="separator"/>
    <addaction name="actionZeitmessung"/>
    <addaction name="separator"/>
    <addaction name="actionBinarisieren"/>
    <addaction name="actionHistogramm"/>
    <addaction name="actionFarbbild_zu_Grauwertbild"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
  </widget>
  <action name="actionBild_laden">
   <property name="text">
    <string>Bild laden</string>
   </property>
  </action>
  <action name="actionEnde">
   <property name="text">
    <string>Ende</string>
   </property>
  </action>
  <action name="actionTestbild_langsam">
   <property name="text">
    <string>Testbild (langsam)</string>
   </property>
  </action>
  <action name="actionTestbild_schnell">
   <property name="text">
    <string>Testbild (schnell)</string>
   </property>
  </action>
  <action name="actionInvertieren">
   <property name="text">
    <string>Invertieren</string>
   </property>
  </action>
  <action name="actionZeitmessung">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zeitmessung</string>
   </property>
  </action>
  <action name="actionScaling">
   <property name="text">
    <string>Skalieren</string>
   </property>
  </action>
  <action name="actionHistogrammausgleich">
   <property name="text">
    <string>Histogrammausgleich</string>
   </property>
  </action>
  <action name="actionKantenfilter_Canny">
   <property name="text">
    <string>Kantenfilter(Canny)</string>
   </property>
  </action>
  <action name="actionLaPlace_Operator">
   <property name="text">
    <string>LaPlace Operator</string>
   </property>
  </action>
  <action name="actionSobelfilter">
   <property name="text">
    <string>Sobelfilter</string>
   </property>
  </action>
  <action name="actionGaussfilter">
   <property name="text">
    <string>Gaussfilter</string>
   </property>
  </action>
  <action name="actionMedianfilter">
   <property name="text">
    <string>Medianfilter</string>
   </property>
  </action>
  <action name="actionDilatieren">
   <property name="text">
    <string>Dilatieren</string>
   </property>
  </action>
  <action name="actionErodieren">
   <property name="text">
    <string>Erodieren</string>
   </property>
  </action>
  <action name="actionBinarisieren">
   <property name="text">
    <string>Binarisieren</string>
   </property>
  </action>
  <action name="actionMittelwertfilter">
   <property name="text">
    <string>Mittelwertfilter</string>
   </property>
  </action>
  <action name="actionFarbbild_zu_Grauwertbild">
   <property name="text">
    <string>Farbbild zu Grauwertbild</string>
   </property>
  </action>
  <action name="actionRotieren">
   <property name="text">
    <string>Rotieren</string>
   </property>
  </action>
  <action name="actionHistogramm">
   <property name="text">
    <string>Histogramm</string>
   </property>
  </action>
  <action name="actionseparierter_Mittelwertfilter">
   <property name="text">
    <string>separierter Mittelwertfilter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
